deployment/postgres-init-db/sql/tables/ui/seen_sources.sql

```diff
@@ -0,0 +1,28 @@
+-- Create a table to track which sources the user has and has not seen added/removed
+create table ui.seen_sources
+(
+    username text not null,
+    external_source_name text not null,
+    external_source_type text not null, -- included for ease of filtering, though not part of pkey
```

Comment:
Missing FK

Reply:
Resolved in [add foreign key to seen_sources.sql](https://github.com/NASA-AMMOS/aerie/commit/6d5e9e57401b14122e92545511bcd0bc6d08e294).

Reply:
The foreign keys are now set up very differently (pointing now to `plan_derivation_group`, following a rethinking of how this feature should work. As such, this comment is now outdated.